internal/common/logmgr/logmgr.go

```diff
@@ -48,7 +48,7 @@ func init() {
 			return function, filenline
 		},
 	}
-	logIns.Level = logrus.InfoLevel
```

Comment:
Thank you for your keen suggestion. This seems very useful. 💯 Overall it seems good to me, and I would like to lastly check out 1 point for my self clarification. 🙏 

If we deprecate `logrus.InfoLevel` and adopt a new feature by providing a new func `GetLogLevel`, do we also have to guide how developers use the customizing the log level to use it? Hope to educate me correctly. 😄

Reply:
Thank you for your additional guidance! It really helps developers to properly use the optional log feature. 😄